### ISSUE TITLE
Cannot select the addressbook where to import.

### ISSUE THREAD

**Issue by maky** [owncloud/contacts#654]
### Steps to reproduce
1. Click import button
-----> Missing step to select whcih addressbook I want to import
2. select the file
3. import it to default addressbook

### Expected behaviour
I have several addressbooks what I share with my family, friends and my partner. Basically this is the only one feature why I use Owncloud, because none of the other products on the market is provide the same.
However after upgrade of 8.2 to 10, I cannot import anymore my other addressbooks. Everything is imported to the same default addressbook. This is pretty bad.

### Server configuration
**Operating system**: linux

**Web server:** apache

**Database:** mysql

**PHP version:** 5.2

**ownCloud version:** 10.0.10

**Contacts version:** 1.5.4

```
No errors have been found.
```

**List of activated apps:**
only market and file
```


**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
yes

#### LDAP configuration (delete this part if not used)

```

**Comment by Julian1998** [comment]
Yeah you are totally right! This is an enhancement I m also missing.

**Comment by Julian1998** [comment]
@DeepDiver1975 That's really annoying. I would suggest we add an occ command for importing adressbooks and calendars to keep it simple for now. What do you think of it?

**Comment by dagroe** [comment]
I added a quick/temporary solution (see a pull request)
